jangjuseong/app.js

```diff
@@ -0,0 +1,51 @@
+// third-party module
+require('dotenv').config(); // 환경변수 사용
```

Comment:
학습을 위한 주석은 앞으로 삭제해주세요!

코드에 대한 설명 목적의 주석은 충분히 의도가 좋은 방향일 수도 있습니다.
하지만, 방대한 양에 대한 코드에 주석을 남긴 후 해당 코드를 수정 할 경우 우리는 주석까지 수정을 신경써서 해줄 필요가 있을 것입니다.

만약 코드가 수정되었는데 주석이 수정이 되지 않았다면, 프로젝트 중반에 투입된 동료 개발자에게 혼선을 야기하는 문제가 있습니다.

코드에 대한 주석을 삭제하고 코드로서 설명할 수 있도록 프로그래밍 하는 것이 최선의 방법이며,
해당 부분 코드는 코드로서도 이미 충분히 설명되는 부분이기 때문에 삭제해도 좋을 것 같습니다!
```suggestion
require('dotenv').config();
```